docker/lint-client

```diff
@@ -15,11 +15,12 @@
 # build stage
 FROM node:lts-alpine as build-stage
 
-RUN apk update && apk add python g++ make && rm -rf /var/cache/apk/*
+RUN apk update && apk add python g++ git make && rm -rf /var/cache/apk/*
 
 WORKDIR /gameroom
 COPY package*.json ./
 
-RUN npm install
+# Need to set as none root user to properly install transact-sdk-javascript
```

Comment:
s/none root/non-root/ (also in the other docker file)

Reply:
fixed